genestack_client/genestack_exceptions.py

```diff
@@ -82,13 +82,18 @@ def __init__(self, current_version, required_version=None):
             message = (
                 'Your Genestack Client version "{current_version}" is too old, '
                 'at least "{required_version}" is required.\n'
-                       ).format(current_version=current_version, required_version=required_version)
+            ).format(current_version=current_version, required_version=required_version)
```

Comment:
Please don't move closing brackets to the same level of indentation as the statement.
(Just the same below)